### PR TITLE
[cache] Bump RedisAdapter default timeout to 5s

### PR DESCRIPTION
**Pull request by Nicofuma**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Bump RedisAdapter timeout to 5s because (at least with Predis) 0 means 0s

### CODE REVIEW

**Comment by Nicofuma** [comment]
There is also an issue with read_timeout as predis uses read_write_timeout (and again, with predis, 0 means 0s), but just renaming the option would be a BC break

File: src/Symfony/Component/Cache/Adapter/RedisAdapter.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function __construct($redisClient, $namespace = '', $defaultLifetime = 0)
      * @param string $dsn
      * @param array  $options See self::$defaultConnectionOptions
      *
-     * @throws InvalidArgumentException When the DSN is invalid.
+     * @throws InvalidArgumentException When the DSN is invalid
```
Comment (HeahDude) [inline src/Symfony/Component/Cache/Adapter/RedisAdapter.php]: This CS fix should target 3.1, so it does not belong to the scope of this PR.

Comment (fabpot) [inline src/Symfony/Component/Cache/Adapter/RedisAdapter.php]: I would just revert this change as fabbot is unfortunately broken right now. PHP-CS-Fixer 2 is doing things we don't want.

**Comment by nicolas-grekas** [comment]
Thank you @Nicofuma.
